serveradmin/servershell/static/js/servershell/autocomplete/command.js

```diff
@@ -117,6 +117,21 @@ $(document).ready(function() {
                         })
                     });
                 }
+                if (
+                    command === 'bookmark' ||
+                    command === 'loadbookmark' ||
+                    command === 'delbookmark'
+                ) {
+                    Object.keys(localStorage).forEach(function(key) {
+                        if (key.startsWith('bookmark.')) {
+                            const name = key.substr(9);
+                            choices.push({
+                                'label': `Bookmark: ${name}`,
+                                'value': `${command} ${name}`,
+                            });
+                        }
```

Comment:
Would just use early return here. Just to reduce some indention for the main code.
```suggestion
                        if (!key.startsWith('bookmark.')) {
                            return
                        }
                        
                        const name = key.substr(9);
                        choices.push({
                            'label': `Bookmark: ${name}`,
                            'value': `${command} ${name}`,
                        });
```